src/main/java/co/rsk/bitcoinj/script/ScriptBuilder.java

```diff
@@ -267,13 +267,12 @@ public static Script createInputScript(@Nullable TransactionSignature signature)
     public static Script createMultiSigOutputScript(int threshold, List<BtcECKey> pubkeys) {
         checkArgument(threshold > 0);
         checkArgument(threshold <= pubkeys.size());
-        checkArgument(pubkeys.size() <= 16);  // That's the max we can represent with a single opcode.
         ScriptBuilder builder = new ScriptBuilder();
-        builder.smallNum(threshold);
+        builder.number(threshold);
```

Comment:
Looks like this class doesn't have any tests. Might be a good idea to add a couple at least for this method. Make sure we can create scripts with more than 15 members

Reply:
https://github.com/rsksmart/bitcoinj-thin/pull/175